### PR TITLE
Update CI

### PR DESCRIPTION
**Pull request by ssiccha**
I've deleted the `latest` tags. We have to use `v1` instead.

### CODE REVIEW

**Comment by wilfwilson** [comment]
There are two more `@latest` tags in the workflow file.

**Comment by ssiccha** [comment]
Fixed, thanks @wilfwilson !

**Review by wilfwilson — APPROVED** [review]

**Comment by cdwensley** [comment]
I find this sort of material very mysterious, but appreciate the benefits of the new testing regime. 
I notice that "latest" appears in the CI.yml of, for example mod, and wonder whether I should be changing all these to "v1"? Advice please!

**Comment by cdwensley** [comment]
Apologies - seems I was wrong about "latest" appearing in xmod.  Must have been looking at an old version.

**Comment by cdwensley** [comment]
Attempted a merge, but now all tests fail: what's going on?

**Comment by cdwensley** [comment]
Seems the testing system is still looking for "latest" - but why?

**Comment by wilfwilson** [comment]
It seems to me that the tests are passing, as you can see the on the Actions tab of the repository: https://github.com/gap-packages/xmodalg/actions. As shown here, the latest commit on the `master` branch shows a green tick, which means that the tests are running as expected (this commit is the one that merged this pull request into the master branch).

Where have you seen a failing test?

**Comment by cdwensley** [comment]
Ah - thanks - I've not been used to looking at the Actions tab, but must remember to do so in future. 
What happened was that, immediately after doing the merge, I received an email from GitHub headed: "[cdwensley/xmodalg] CI workflow run", followed by "CI: All jobs have failed", and then a list starting with "(X) CI / master - ubuntu-latest Failed in 52 seconds".  Obviously best ignored?

**Comment by wilfwilson** [comment]
> I find this sort of material very mysterious, but appreciate the benefits of the new testing regime.
> I notice that "latest" appears in the CI.yml of, for example mod, and wonder whether I should be changing all these to "v1"? Advice please!

It's understandable that you find it mysterious. But I hope you're not too worried by it, and if you're happy to trust us you can ignore its details.

There's nothing particularly special in the GitHub Actions testing setup at Xmodalg: the setup that has been added to Xmodalg and to most other GAP packages (predominantly by @fingolfin and @ssiccha, and a little bit of me), is pretty much the same as that used in the [Example package](https://github.com/gap-packages/example).

The aim is to have a testing system that is as standard as possible across the packages, in order to make its installation and maintenance as easy and efficient as possible, while still being useful. In particular, it allows package authors/maintainers to benefit from an automated testing setup without having to learn its technical details.

The system is mostly settled in now, but we're still learning how to do things best, and so it's possible that there'll be a couple of (possibly automated) pull requests to update the system in the coming months. Essentially, these would be to bring Xmodalg into line with improvements/fixes made to the testing system in the Example package.

**Comment by wilfwilson** [comment]
Ah, @cdwensley – the failure occurred on your fork of the repository (i.e. `cdwensley/xmodalg`, not here, which is `gap-packages/xmodalg`). Tests run on both repositories, depending on what is pushed where.

You can see the Actions tab on your fork of the repository here: https://github.com/cdwensley/xmodalg/actions

And thankfully everything is fine! We can see what happened: the first run of the tests failed: those tests were run on commit 124603a on the `master` branch, which was before this PR #44 was merged in. The purpose of this PR was to fix exactly this problem. But then you can see that in the subsequent run of the tests, which was on commit 2024bca of the `master` branch, and which was after PR #44 was merged, that the tests are now passing.

So in the future, the tests should continue to pass on subsequent pull requests and pushes (unless something else breaks).

**Comment by wilfwilson** [comment]
In my opinion, when you get GitHub emails about such failures, it's a good idea to at least briefly investigate what happened by following the link in the email. Hopefully this won't happen very often.

If there has been a subsequent run of the tests on the same branch which has passed, then it's probably safe to ignore it as a transient problem. Otherwise, there's probably a deeper problem that should be investigated and fixed - the purpose of this automation is to alert us to such problems.

**Comment by cdwensley** [comment]
Many thanks for all your explanation and advice.  I have made a note of it.

**Pull request merged**
